rs/interfaces/src/p2p/consensus.rs

```diff
@@ -145,3 +124,26 @@ pub trait ArtifactAssembler<A1: IdentifiableArtifact, A2: PbArtifact>:
         peers: P,
     ) -> impl std::future::Future<Output = Result<(A1, NodeId), Aborted>> + Send;
 }
+
+/// The artifact filter function returns a value that defines 3 possible handling logics when
```

Comment:
I'm not sure `FilterValue` is much better than `Priority`.  Filtering something is usually associated with either removing or keeping something.

Reply:
I agree, filter is not ideal. But I could not come up with a better word. I am open for suggestions. 

However, I do think it is better than priority. Priority defines order where a filter is more of a admission policy.

Reply:
> I agree, filter is not ideal. (...)
> However, I do think it is better than priority. 

Agree to both.

> (...)  I am open for suggestions.

How about something like 
```
enum Needed {
   /// The client needs the artifact now.
   Now,
   /// The client might need the artifact later.
   MaybeLater,
   /// The client will never need the artifact.
   Never,
}
```
I'm not sure about the enum name but I like the variant names

Reply:
How about BouncerValue, BouncerFilter, etc.

Because this is exactly what a bouncer at a night club will do :) 

The variant names I also like, I would prefer to keep them as is